src/components/Player/Opsheet/Opsheet.js

```diff
@@ -1,17 +1,55 @@
 import React, { Component } from 'react';
-import List from './List'
-import Toggle from './Toggle'
+import List from './List';
+import ListType from './ListType';
+
+//function to convert to react
+// function collapseToggle () {
+//   let collapsed = true;
+//   const toggle = document.getElementsByClassName('collapse-toggle')
+//   const opsheetContent = document.getElementsByClassName('list-wrapper')
+  
+//   toggle[0].addEventListener('click', function() {
+//     if (collapsed === true) {
+//       opsheetContent[0].style.display = 'block';
+//       collapsed = false
+//     } else {
+//       opsheetContent[0].style.display = 'none';
+//       collapsed = true
+//     }
+//   })
+// }
 
 class Opsheet extends Component {
+  state = { 
+    collapsed: true, 
+  };
+
+  collapseSheet = () => {
+    if (this.state.collapsed === true){
+      this.setState({
+        collapsed: false,
+      })
+    } else {
+      this.setState({
+        collapsed: true,
+      })
+    }
```

Comment:
Consider:

```js
collapseSheet = () => {
    this.setState({collapsed: !this.state.collapsed});
}
```

Also, I think "toggle" is a better name for this functionality; maybe `toggleSheetCollapseExpand`, since it handles both collapsing and expanding.